### PR TITLE
DOC-2225: TinyMCE 6.8.2 release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2225

Changes:
* added `6.8.2-release-notes.adoc` to project; updated `changelog.adoc`, `nav.adoc` and `release-notes.adoc` for the TinyMCE 6.8.2 release.
* removal of `duplication` entry notes in the `changelog.adoc` for `6.7.2` release.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+The following premium plugin updates were released alongside {productname} 6.8.2.
+
+=== PowerPaste 6.2.4
+
+The {productname} 6.8.2 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** 6.2.4 includes the following fix.
+
+==== `white-space-collapse` CSS property was not correctly supported when pasting from Google Docs.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:29]: @mkzhx should we add links to [mdn-css_properties_white-space-collapse](https://caniuse.com/mdn-css_properties_white-space-collapse) and the below for this?

- [#white-space-property](https://www.w3.org/TR/css-text-3/#white-space-property)
- [#white-space-collapsing](https://www.w3.org/TR/css-text-4/#white-space-collapsing)

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+=== Clicking inside table cells with heavily nested content could cause the browser to hang.
+// #TINY-10380
+Previously when clicking below content that is heavily nested inside a block such as a table cell, would cause the browser to hang.
+
+{productname} 6.8.2 addresses this, by optimizing the code responsible for the hang while updating the algorithm.
+
+As a result, the browser no longer hangs, when clicking inside table cells with heavily nested content.
+
+=== Toggling a list that contains an LI element having another list as its first child would remove the remaining content within that LI element.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:66]: @lorenzo-pomili Please review

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  <ul>...</ul>
+  <something-else/>
+</li>
+----
+
+As a consequence, any tags nested within the `li` would be deleted from the {productname} document, resulting in content loss for the user.
+
+{productname} 6.8.2 addresses this by improving the representation of the list to manage this case.
+
+As a result, now, when toggling `<li>` elements using the `bullist` button, content within a list is now correctly recreated the content is retained as expected.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:83]: @lorenzo-pomili please review this section for the 6.8.2 release notes.

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+As of {productname} 6.8.2, PowerPaste now supports working with the shorthand `white-space` property by removing the expanded `white-space-collapse` and `text-wrap` properties when present from Google Docs content.
+
+For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].
+
+
+[[bug-fixes]]
+== Bug fixes
+
+{productname} 6.8.2 also includes the following bug fix<es>:
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:41]:
```suggestion
{productname} 6.8.2 also includes the following bug fixes:
```

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+// #TINY-10380
+Previously when clicking below content that is heavily nested inside a block such as a table cell, would cause the browser to hang.
+
+{productname} 6.8.2 addresses this, by optimizing the code responsible for the hang while updating the algorithm.
+
+As a result, the browser no longer hangs, when clicking inside table cells with heavily nested content.
+
+=== Toggling a list that contains an LI element having another list as its first child would remove the remaining content within that LI element.
+// #TINY-10414
+In a previous xref:6.7.2-release-notes.adoc#toggling-a-list-that-contained-a-list-item-element-li-which-in-turn-contained-another-list-item-element-as-its-first-child-removed-other-content-within-the-first-list-item-element[fix] _(toggling-a-list-that-contained-a-list-item-element-li-which-in-turn-contained-another-list-item-element-as-its-first-child-removed-other-content-within-the-first-list-item-element)_ for lists, some element structures would not recreate the `<li>` tags when a list element was toggled using the `bullist` button in the toolbar.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:74]:
```suggestion
In a previous xref:6.7.2-release-notes.adoc#toggling-a-list-that-contained-a-list-item-element-li-which-in-turn-contained-another-list-item-element-as-its-first-child-removed-other-content-within-the-first-list-item-element[bugfix] for lists, some element structures would not recreate the `<li>` tags when a list element was toggled using the `bullist` button in the toolbar.
```

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+.Example structure 
+[source, html]
+----
+<li>
+  <ul>...</ul>
+  <something-else/>
+</li>
+----
+
+As a consequence, any tags nested within the `li` would be deleted from the {productname} document, resulting in content loss for the user.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:85]:
```suggestion
As a consequence, any tags nested within the `<li>` would be deleted from the {productname} document, resulting in content loss for the user.
```

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+As of {productname} 6.8.1, the accessible label of the `align` bespoke select toolbar button announced both the name of the button and the currently selected value, for example “Block Paragraph”. However, corresponding translation strings with placeholders were not added. This resulted in the current value being translated as expected, but not the name of the button. 
+
+In {productname} 6.8.2, new translation strings were added to ensure names of the bespoke select toolbar buttons are translated as well, making their accessible labels appropriately translated.
+
+=== Clicking inside table cells with heavily nested content could cause the browser to hang.
+// #TINY-10380
+Previously when clicking below content that is heavily nested inside a block such as a table cell, would cause the browser to hang.
+
+{productname} 6.8.2 addresses this, by optimizing the code responsible for the hang while updating the algorithm.
```
Comment (spocke) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:68]:
```suggestion
{productname} 6.8.2 addresses this, by optimizing algorithm that was causing the browser to hang.
```

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/6.8.2-release-notes.adoc (hunk 1 above)
Comment (mkzhx) [reply modules/ROOT/pages/6.8.2-release-notes.adoc:29]: Yes that's a good idea, I think just the W3 spec links are sufficient.

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+// #TINY-10435
+
+As of {productname} 6.8.1, the accessible label of the `align` bespoke select toolbar button announced both the name of the button, “Align”, and the currently selected value. This would result in the accessible label of the select button to be “Align left”, “Align right”, “Align center”, or “Align justify” depending on the value selected. However, the problem with the first three labels is that they are the same as the normal align toolbar buttons. This is misleading as unlike the normal align buttons, the bespoke button does not result in the aligning action, and instead opens a dropdown. The problem with “Align justify” is that the two verbs do not make grammatical sense in terms of describing the text alignment.
+
+As of {productname} 6.8.2, the name of the `align` bespoke select button is changed to “Alignment”, followed by the currently selected value. This results in the accessible labels being “Alignment left”, “Alignment right”, “Alignment center”, or “Alignment justify” depending on the values selected, which more clearly indicates the currently selected text alignment.
```
Comment (mkzhx) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:55]:
```suggestion
As of {productname} 6.8.2, the name of the `align` bespoke select button is changed to “Alignment”, followed by the currently selected value. This results in the accessible labels being “Alignment left”, “Alignment right”, “Alignment center”, or “Alignment justify” depending on the value selected, which more clearly indicates the currently selected text alignment.
```

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+=== Accessible names of bespoke select toolbar buttons including `align`, `fontfamily`, `fontsize`, `blocks`, and `styles` were incorrectly translated.
+// #TINY-10426 #TINY-10435
+
+As of {productname} 6.8.1, the accessible label of the `align` bespoke select toolbar button announced both the name of the button and the currently selected value, for example “Block Paragraph”. However, corresponding translation strings with placeholders were not added. This resulted in the current value being translated as expected, but not the name of the button.
```
Comment (mkzhx) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:60]:
```suggestion
As of {productname} 6.8.1, the accessible labels of the bespoke select toolbar buttons announced both the name of the button and the currently selected value, for example “Block Paragraph”. However, corresponding translation strings with placeholders were not added. This resulted in the current value being translated as expected, but not the name of the button. 
```

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+
+As of {productname} 6.8.2, the name of the `align` bespoke select button is changed to “Alignment”, followed by the currently selected value. This results in the accessible labels being “Alignment left”, “Alignment right”, “Alignment center”, or “Alignment justify” depending on the values selected, which more clearly indicates the currently selected text alignment.
+
+=== Accessible names of bespoke select toolbar buttons including `align`, `fontfamily`, `fontsize`, `blocks`, and `styles` were incorrectly translated.
+// #TINY-10426 #TINY-10435
+
+As of {productname} 6.8.1, the accessible label of the `align` bespoke select toolbar button announced both the name of the button and the currently selected value, for example “Block Paragraph”. However, corresponding translation strings with placeholders were not added. This resulted in the current value being translated as expected, but not the name of the button. 
+
+In {productname} 6.8.2, new translation strings were added to ensure names of the bespoke select toolbar buttons are translated as well, making their accessible labels appropriately translated.
```
Comment (mkzhx) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:62]:
```suggestion
In {productname} 6.8.2, new translation strings were added to ensure names of the bespoke select toolbar buttons are translated as well, making their accessible labels appropriately internationalized.
```

**Review by mkzhx — APPROVED** [review]

File: modules/ROOT/pages/6.8.2-release-notes.adoc [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+  <ul>...</ul>
+  <something-else/>
+</li>
+----
+
+As a consequence, any tags nested within the `<li>` would be deleted from the {productname} document, resulting in content loss for the user.
+
+{productname} 6.8.2 addresses this by improving the representation of the list to manage this case.
+
+As a result, now, when toggling `<li>` elements using the `bullist` button, content within a list is now correctly recreated the content is retained as expected.
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.2-release-notes.adoc:89]: @lorenzo-pomili waiting for review on this section.

**Review by lorenzo-pomili — APPROVED** [review]

File: modules/ROOT/pages/6.8.2-release-notes.adoc (hunk 11 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/6.8.2-release-notes.adoc:89]: approved

**Pull request merged**